### DISCUSSION TITLE
Lesson 3 (Fund Me) : Gas Estimation keeps failing

### DISCUSSION THREAD

**Discussion by Noveleader** [smartcontractkit/full-blockchain-solidity-course-py#1182] [Q&A] [answered]
Never Mind got the answer, didn't search well people already faced this problem earlier.
Thanks

**Comment by cromewar** [comment, marked as answer]
Hello @Noveleader 
Please if you already have posted the problem do not edit it, keep it as it is as this could be helpful to other users.
Anyways take this in consideration for future questions.
Cheers.

Closing this thread now.
